tensornetwork/matrixproductstates/mps.py

```diff
@@ -371,7 +371,16 @@ def __init__(self,
       raise ValueError(
           'center_position = {} not between 0 <= center_position < {}'.format(
               center_position, len(tensors)))
-    self.center_position = center_position
+    if center_position == 0:
```

Comment:
Yes, making it harder for people to shoot themselves in the foot seems like a good default :)
We can always add a flag to skip this step later, if we want.